deployments/get/objects.go

```diff
@@ -387,6 +387,7 @@ func GenerateDaemonSet(env, namespace string) *appsv1.DaemonSet {
 	var terminationGracePeriodSeconds = int64(30)
 	var args = []string{
 		"-gRPC=" + strconv.Itoa(int(port)),
+		"-lsm=" + lsmOrder,
```

Comment:
Should we add this? We handle `defaultPosture` arguments on the client side, Ref https://github.com/kubearmor/kubearmor-client/blob/ba6ce813f637680113f7d5908bc0b40820816624/install/install.go#L210

Reply:
Agree, waiting for this to be merged to update karmor https://github.com/kubearmor/KubeArmor/issues/788#issuecomment-1247815039

Reply:
What I meant is we should patch the deployment after generating? Instead of modifying the Generate function. Like we are doing for defaultPosture.

Reply:
Undrestood.